course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -286,14 +286,26 @@
             // return the filtered list
             return filteredResults;
         };
+
+        $scope.getCourseDescription = function(course) {
```

Comment:
Yeah I guess this could go either way -- it's less important on the people page as it's just in that info box at the top, and we already know what context we're in by the course instance ID, for example. But probably nice to have it in the box, so I guess leave it?